app/gui/view/debug_scene/cloud-dashboard/src/projects_table.rs

```diff
@@ -0,0 +1,698 @@
+//! Module containing the [`ProjectsTable`], which is used to display a list of [`Project`]s in the
+//! Cloud dashboard.
+//!
+//! The list of [`Project`]s, as rendered, contains information about the state of the [`Project`],
+//! the data associated with the [`Project`], the permissions assigned to it, etc. Buttons for
+//! interacting with the [`Project`]s (e.g., starting and stopping the [`Project`]s) are also
+//! rendered.
+//!
+//! [`Project`]: ::enso_cloud_view::project::Project
+
+use ensogl::prelude::*;
+
+use enso_cloud_view as view;
+use ensogl::application;
+use ensogl::data::color;
+use ensogl_core::application::Application;
+use ensogl_core::display;
+use ensogl_core::frp;
+use ensogl_grid_view::simple::EntryModel;
+use ensogl_grid_view::Col;
+use ensogl_grid_view::Row;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+/// The width of a single entry in the [`ProjectsTable`], in pixels.
+// FIXME [NP]: https://www.pivotaltracker.com/story/show/183909467
+// Move the style constants to StyleWatchFrp variables.
+const ENTRY_WIDTH: f32 = 130.0;
+/// The height of a single entry in the [`ProjectsTable`], in pixels.
+// FIXME [NP]: https://www.pivotaltracker.com/story/show/183909467
+// Move the style constants to StyleWatchFrp variables.
+const ENTRY_HEIGHT: f32 = 28.0;
+/// The grid is intended to take up 20% of the viewport height, expressed as a fraction;
+// FIXME [NP]: https://www.pivotaltracker.com/story/show/183909467
+// Move the style constants to StyleWatchFrp variables.
```

Comment:
If you are creating a fixme, in multiple lines, these lines need to be indented in rder for IDEs like Intellij to understand that. Please indent these.